esp-hal/src/sdio.rs

```diff
@@ -0,0 +1,12 @@
+//! # Secure Digital I/O - Slave Mode
+//!
+//! ## Overiew
+//!
+//! The peripheral can be used to transfer data over the SDIO bus in `Slave` mode.
+
+#[cfg(feature = "esp32c6")]
```

Comment:
It's likely this isn't needed based on my other comments.

Reply:
You were right, I refactored taking your comments into account, and this is no longer needed.